src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -1480,38 +1491,77 @@ && isAwaitingAnotherDefendantResponse(caseData)) {
     }
 
     private void assembleResponseDocumentsSpec(CaseData caseData, CaseData.CaseDataBuilder<?, ?> updatedCaseData) {
```

Comment:
See how much of the refactor suggested in RespondToClaimCallbackHandler is applicable here. They seem very similar to me. Maybe it's worth extracting the commonalities to a base abstract class

Reply:
Discussed, refactoring would be a nice have, but not part of this ticket/PR, potential tech debt ticket

Reply:
Agreed. Marking as resolved.